jans-linux-setup/jans_setup/app_info.json

```diff
@@ -16,5 +16,5 @@
   "CRYPTOGRAPHY": "https://files.pythonhosted.org/packages/20/8b/66600f5851ec7893ace9b74445d7eaf3499571b347e339d18c76c876b0f9/cryptography-37.0.4-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl",
   "TWILIO_MAVEN": "https://repo1.maven.org/maven2/com/twilio/sdk/twilio/",
   "TWILIO_VERSION": "7.17.0",
-  "KC_VERSION": "22.0.3"
+  "KC_VERSION": "23.0.1"
```

Comment:
Please bump the version to 23.0.3 , not 23.0.1

Reply:
Done https://github.com/JanssenProject/jans/pull/7140/commits/a8abb7bea00b64ecb0720ad5b3a00788cfaf604f